src/use-case/question-answering.md

```diff
@@ -0,0 +1,257 @@
+# Generative Question-Answering
+Vector search can also be used to provide context to generative models such as OpenAI's GPT series, improving the quality of the outputs. 
+
+This post will explore the use of `pgvecto.rs` in building and retrieving from a vector knowledge base to bolster the performance of generative models in question-answering tasks. 
```

Comment:
Could you introduce a bit about what the knowledge base in the example is?

Reply:
Do you mean explain more about what a knowledge base is in general, or more specifically the movies knowledge base for this example?

Reply:
The latter. I think.